### ISSUE TITLE
chef_omnibus_install_options not appended properly

### ISSUE THREAD

**Issue by tknerr** [test-kitchen/test-kitchen#580]
Given a configuration like this:
```yaml
---
driver:
  name: vagrant

provisioner:
  name: chef_solo
  require_chef_omnibus: 11.16.4
  chef_omnibus_install_options: -d /tmp/vagrant-cache/vagrant_omnibus

...
```

The omnibus installation fails with an error like this:
```
...

if should_update_chef "/opt/chef" "11.16.4" ; then
  echo "-----> Installing Chef Omnibus (11.16.4)"
  do_download https://www.chef.io/chef/install.sh /tmp/install.sh
  sudo -E sh /tmp/install.sh -v 11.16.4-d /tmp/vagrant-cache/vagrant_omnibus
else
  echo "-----> Chef Omnibus installation detected (11.16.4)"
fi
']
>>>>>> ----------------------
rake aborted!
Command failed with status (10): [kitchen test --log-level info...]
/home/tkn/sample-toplevel-cookbook/Rakefile:28:in `block in <top (required)>'
Tasks: TOP => integration
(See full trace by running task with --trace)

```

Workaround: enforce the missing space inbetween the version and install options:
```yaml
provisioner:
  name: chef_solo
  require_chef_omnibus: 11.16.4
  chef_omnibus_install_options: ' -d /tmp/vagrant-cache/vagrant_omnibus'
```

**Comment by fnichol** [comment]
Very sorry for this, I have a fix in #581 and will merge-and-release once Travis gives me the green thumbs up.

**Comment by tknerr** [comment]
@fnichol no way to be sorry, thanks for the quick fix & release! :+1: :+1: :+1:
